w/wazuh-api/wazuh-api_ubi_8.7.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package          : wazuh-api
+# Version          : v3.13.6
+# Source repo      : https://github.com/wazuh/wazuh-api
+# Tested on        : UBI 8.7
+# Language         : Javascript
+# Travis-Check     : True
+# Script License   : GNU General Public License v2.0
+# Maintainer       : Vinod K <Vinod.K1@ibm.com>
+#
+# Disclaimer       : This script has been tested in root mode on given
+# ==========         platform using the mentioned version of the package.
+#                    It may not work as expected with newer versions of the
+#                    package and/or distribution. In such case, please
+#                    contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=wazuh-api
+PACKAGE_VERSION=${1:-v3.13.6}
+PACKAGE_URL=https://github.com/wazuh/wazuh-api
+
+HOME_DIR=${PWD}
+
+OS_NAME=$(grep ^PRETTY_NAME /etc/os-release | cut -d= -f2)
+
+yum install -y git wget curl make sudo cmake gcc gcc-c++ autoconf procps diffutils libffi-devel sqlite-libs sqlite-devel python38 python38-devel python3-policycoreutils automake libtool openssl-devel yum-utils libstdc++-static  hostname
+
+NODE_VERSION=v18.9.0
+cd /
```

Comment:
@vinodk99 Can we operate in home directory or default location only. So it will not create issue when run this build through CI in currency

Reply:
@nikhil-kalbande ,updated the script as per your suggestion.